### ISSUE TITLE
Ubuntu 20.04 arm64 packages fail to install as of today

### ISSUE THREAD

**Issue by glebm** [actions/runner-images#7902]
### Description

Today, our Aarch64 cross build workflow failed with the following error:

```
$ sudo apt-get install -y cmake git smpq gettext crossbuild-essential-arm64 libsdl2-dev:arm64 libsdl2-image-dev:arm64 libsodium-dev:arm64 libsimpleini-dev:arm64 libpng-dev:arm64 libbz2-dev:arm64 libfmt-dev:arm64

...

Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
```

This workflow has worked fine until now.

* Workflow file: https://github.com/diasurgical/devilutionX/actions/runs/5525336672/workflow
* Failed run: https://github.com/diasurgical/devilutionX/actions/runs/5525336672/jobs/10078831837

/cc @AJenbo

### Platforms affected

- [ ] Azure DevOps
- [x] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [X] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

20230702.1.0

### Is it regression?

yes, same version, https://github.com/diasurgical/devilutionX/actions/runs/5517420608/jobs/10060014683

### Expected behavior

sucess

### Actual behavior

failure

### Repro steps

Run the workflow, or simply run this:

``` bash
# Work around the somewhat broken packages in the GitHub Actions Ubuntu 20.04 image.
# https://github.com/actions/runner-images/issues/4620#issuecomment-981333260
sudo apt-get -y install --allow-downgrades libpcre2-8-0=10.34-7
# Install arm64 cross-build packages
Packaging/nix/debian-cross-aarch64-prep.sh
```

(https://github.com/diasurgical/devilutionX/blob/master/Packaging/nix/debian-cross-aarch64-prep.sh)

**Comment by ilia-shipitsin** [comment]
Hello, @glebm 

I see "apt-get install ..." in workflow (but no "apt-get update")
https://github.com/diasurgical/devilutionX/actions/runs/5525336672/workflow#L32-L37

"apt-get install ..." operates with local package cache, which should be updated first, for example: https://blog.packagecloud.io/you-need-apt-get-update-and-apt-get-upgrade/

**Comment by AJenbo** [comment]
Apt update didn't help when I tried it earlier

**Comment by glebm** [comment]
@ilia-shipitsin Hi Ilia, that's not the step that fails. We do invoke `sudo apt-get update`:

https://github.com/diasurgical/devilutionX/blob/de081b7080ada9814966682f8357828a9aedf31d/Packaging/nix/debian-cross-aarch64-prep.sh#L21

**Comment by glebm** [comment]
The issue got fixed in the new image version (20230710.1.0) https://github.com/diasurgical/devilutionX/actions/runs/5537840856/jobs/10107118804
